Sources_v3/StreamChatUI/ChatChannel/MessageComposerSuggestionsViewController.swift

```diff
@@ -4,6 +4,40 @@
 
 import UIKit
 
+public struct MessageComposerSuggestionCommand {
+    let name: String
+    let hintText: String
+    let onCommandTap: () -> Void
+    let icon: UIImage?
+
+    public static var giphy: MessageComposerSuggestionCommand {
+        MessageComposerSuggestionCommand(
+            name: "Giphy",
+            hintText: "/giphy [query]",
+            onCommandTap: {},
+            icon: MessageComposerSuggestionCommandKind.giphy.image
+        )
+    }
+
+    public static var imgur: MessageComposerSuggestionCommand {
+        MessageComposerSuggestionCommand(
+            name: "Imgur",
+            hintText: "/imgur [query]",
+            onCommandTap: {},
+            icon: MessageComposerSuggestionCommandKind.imgur.image
+        )
+    }
+
+    public static var customCommand: MessageComposerSuggestionCommand {
+        MessageComposerSuggestionCommand(
+            name: "Some Command",
+            hintText: "/command",
+            onCommandTap: {},
+            icon: MessageComposerSuggestionCommandKind.custom(identifier: "").image
+        )
+    }
```

Comment:
This should be loaded from the channel config

Reply:
Fixed.